examples/connector/python/main.py

```diff
@@ -0,0 +1,139 @@
+import grpc
+from concurrent import futures
+import json
+import sys
+sys.path.append('sdk_pb2')
+
+from sdk_pb2 import connector_sdk_pb2_grpc
+from sdk_pb2 import common_pb2
+from sdk_pb2 import connector_sdk_pb2
+
+
+class ConnectorService(connector_sdk_pb2_grpc.ConnectorServicer):
+    def ConfigurationForm(self, request, context):
+        form_fields = common_pb2.ConfigurationFormResponse(schema_selection_supported=True,
+                                                           table_selection_supported=True)
+        form_fields.fields.add(name="apiKey", label="API Key", required=True, text_field=common_pb2.TextField.PlainText)
+        form_fields.fields.add(name="password", label="User Password", required=True, text_field=common_pb2.TextField.Password)
+
+        form_fields.fields.add(
+            name="region",
+            label="AWS Region",
+            required=False,
+            dropdown_field=common_pb2.DropdownField(dropdown_field=["US-EAST", "US-WEST"])
+        )
+
+        form_fields.fields.add(name="hidden", label="my-hidden-value", text_field=common_pb2.TextField.Hidden)
+        form_fields.fields.add(name="isPublic", label="Public?", description="Is this public?", toggle_field=common_pb2.ToggleField())
+
+        # add setup tests
+        form_fields.tests.add(name="connection_test", label="Tests connection")
+
+        return form_fields
+
+    def Test(self, request, context):
+        configuration = request.configuration
+        # Name of the test to be run
+        test_name = request.name
+        print("Configuration: ", configuration)
+        print("Test name: ", test_name)
+        return common_pb2.TestResponse(success=True)
+
+    def Schema(self, request, context):
+        table_list = common_pb2.TableList()
+        t1 = table_list.tables.add(name="table1")
+        t1.columns.add(name="a1", type=common_pb2.DataType.UNSPECIFIED, primary_key=True)
+        t1.columns.add(name="a2", type=common_pb2.DataType.DOUBLE)
+
+        t2 = table_list.tables.add(name="table2")
```

Comment:
This example doesn't use `table2`. Can you add an upsert for it so it gets a row in the destination?

Reply:
Copied from the java connector example, even it does not use it table2. I'll add simple upsert operation for it.